<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ADC</class>
 <widget class="QWidget" name="ADC">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>234</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLCDNumber" name="adc_lcdNumber">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>361</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="adc_startButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QPushButton" name="adc_stopButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>160</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>结束</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
